helm/cluster-cloud-director/Chart.yaml

```diff
@@ -13,6 +14,7 @@ keywords:
   - vcd
 annotations:
   application.giantswarm.io/team: "rocket"
+  ui.giantswarm.io/logo: https://s.giantswarm.io/app-icons/vmware-cloud-director/1/logo_light.png
```

Comment:
Gives 404?

Reply:
The PR with team-rainbow has not been merged yet

Reply:
it's 200 now